pkg/statsd/handler_fixtures_test.go

```diff
@@ -50,6 +50,41 @@ func (nh *nopHandler) DispatchEvent(ctx context.Context, e *gostatsd.Event) {
 func (nh *nopHandler) WaitForEvents() {
 }
 
+type expectingHandler struct {
+	countingHandler
+	
+	wgMetrics    sync.WaitGroup
```

Comment:
Is it worth using @ash2k stepper library.

Reply:
Nah, ordering doesn't matter here, it's just about "all finished".  I did consider using a single WG, but that wouldn't count things.

It is kinda bad because there's no timeout though.